niworkflows/interfaces/bids.py

```diff
@@ -279,7 +279,8 @@ class _DerivativesDataSinkInputSpec(DynamicTraitedSpec, BaseInterfaceInputSpec):
         File(exists=True), mandatory=True, desc="the object to be saved"
     )
     meta_dict = traits.DictStrAny(desc="an input dictionary containing metadata")
-    source_file = File(exists=False, mandatory=True, desc="the input func file")
+    source_file = InputMultiObject(
```

Comment:
one thing to note: we might want to deprecate `source_file` and replace it with `source_files`